platform-includes/set-fingerprint/basic/native.mdx

```diff
@@ -1,3 +1,15 @@
+You can set a fingerprint to be added to all future events and errors:
+```c
+#include <sentry.h>
+
+sentry_set_fingerprint("myFingerPrint", NULL); // overwrites existing fingerprints
```

Comment:
Do we need to document here why the `NULL` is needed? It's already described in the function's docstring in the SDK:
>  This accepts a variable number of arguments, and needs to be terminated by a trailing `NULL`.

For context, this is just a C-specific way to close out a list of variadic arguments.